### PR TITLE
Small fixes after guides review

### PR DESCRIPTION
**Pull request by radtriste**
Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributions doc](https://github.com/kiegroup/kogito-docs/blob/main/CONTRIBUTING.md)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [ ] The nav.adoc file has a link to this guide in the proper category
- [ ] The index.adoc file has a card to this guide in the proper category, with a meaningful description

### CODE REVIEW

**Review by ricardozanini — APPROVED** [review]

**Review by spolti — APPROVED** [review]

File: serverlessworkflow/modules/ROOT/pages/service-orchestration/orchestration-of-openapi-based-services.adoc [hunk 1]
```diff
@@ -123,7 +119,7 @@ To send requests with the REST calls, first you need to define the OpenAPI funct
 +
 Once you have access to the OpenAPI specification files, you can define the functions in the Serverless Workflow. 
 
-You can also use the link:{kogito_sw_examples_url}/serverless-workflow-temperature-conversion[`serverless-workflow-temperature-conversion`] example application as a reference.
+You can also use the link:{kogito_sw_examples_url}/serverless-workflow-temperature-conversion[serverless-workflow-temperature-conversion] example application as a reference.
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/service-orchestration/orchestration-of-openapi-based-services.adoc:122]: For consistency, the example applications are written as code. 
```suggestion
You can also use the link:{kogito_sw_examples_url}/serverless-workflow-temperature-conversion[`serverless-workflow-temperature-conversion`] example application as a reference.
```

File: serverlessworkflow/modules/ROOT/pages/service-orchestration/orchestration-of-openapi-based-services.adoc [hunk 2]
```diff
@@ -207,6 +203,11 @@ components:
 
 <1> Function definition for OpenAPI multiplication function
 <2> Function definition for OpenAPI subtraction function
+
+[NOTE]
+====
+Note that in the above example, we are storing the `multiplication` and `substraction` OpenAPI files into the `src/main/resources/specs` folder of our application.
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/service-orchestration/orchestration-of-openapi-based-services.adoc:209]:
```suggestion
Note that in the previous example, the `multiplication` and `subtraction` OpenAPI files are stored in the `src/main/resources/specs` folder of the application.
```

**Review by hmanwani-rh — COMMENTED** [review]
@radtriste Added a couple of suggestions. The rest looks good.

File: serverlessworkflow/modules/ROOT/pages/service-orchestration/orchestration-of-openapi-based-services.adoc (hunk 1 above)
Comment (radtriste) [reply serverlessworkflow/modules/ROOT/pages/service-orchestration/orchestration-of-openapi-based-services.adoc]: ok...
@hmanwani-rh the main reason I wanted to change this is that in the documentation, with this formatting, we don't make the distinction between a example application mention and a link ...

- `link:{kogito_sw_examples_url}/serverless-workflow-temperature-conversion[`serverless-workflow-temperature-conversion`]`
- `serverless-workflow-temperature-conversion`

both appears as same formatting in the documentation and we may not know that we can click on it ...

Comment (hmanwani-rh) [reply serverlessworkflow/modules/ROOT/pages/service-orchestration/orchestration-of-openapi-based-services.adoc]: +1 on the formatting. 
Maybe we can change the style after DP1 to show the code links with an underline. 

CC @ricardozanini

Comment (radtriste) [reply serverlessworkflow/modules/ROOT/pages/service-orchestration/orchestration-of-openapi-based-services.adoc]: do you need an issue for that ?

Comment (hmanwani-rh) [reply serverlessworkflow/modules/ROOT/pages/service-orchestration/orchestration-of-openapi-based-services.adoc]: Sure, we can create one.

**Pull request merged**